src/lib/logical_query_plan/abstract_lqp_node.hpp

```diff
@@ -264,6 +266,20 @@ class AbstractLQPNode : public std::enable_shared_from_this<AbstractLQPNode>, pr
    * @}
    */
 
+  /**
+   * @defgroup Comparing two LQPs
+   * shallow_equals() compares only the nodes without considering the children, find_subplan_mismatch() will compare the entire
+   * sub plan
+   * @}
```

Comment:
this should be `{` and the one below `}` ;)

Reply:
yep